openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -111,6 +111,8 @@ trait JsonSchemas extends endpoints.algebra.JsonSchemas {
 
   lazy val booleanJsonSchema: DocumentedJsonSchema = Primitive("boolean")
 
+  lazy val byteJsonSchema: DocumentedJsonSchema = Primitive("byte")
```

Comment:
There is no `byte` type in OpenAPI. We should just use `integer`. Unfortunately, there is no `int8` *format* (as there is `int32` for `Int` values).